src/components/Forms.js

```diff
@@ -19,21 +19,30 @@ const NewBookForm = () => {
   };
 
   return (
-    <form onSubmit={handleFormSubmit}>
-      <input
-        type="text"
-        name="title"
-        placeholder="Title"
-        required
-      />
-      <input
-        type="text"
-        name="author"
-        placeholder="Author"
-        required
-      />
-      <button type="submit">Add New Book</button>
-    </form>
+    <div className="book-input">
+      <div className="hr" />
+      <h3 className="addTitle Text-Style-10">
+        ADD NEW BOOK
+      </h3>
+      <form onSubmit={handleFormSubmit} className="form-input">
+
+        <input
+          type="text"
+          name="title"
+          placeholder="Title"
+          className="form-input"
+          required
+        />
+        <input
+          type="text"
+          name="author"
+          placeholder="Author"
+          className="form-input"
+          required
+        />
+        <button type="submit" className="add-btn">ADD BOOK</button>
+      </form>
+    </div>
```

Comment:
- [ ] Kindly fix the input fields positioning and alignment by giving the correct margin values to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/230731414-89ea7a00-837f-44fa-8d89-cf49a2bb1f8f.png) | ![image](https://user-images.githubusercontent.com/68796232/230731408-e01a9b00-307f-474b-b82e-ac4236e6630b.png)  |